en/docs/learn/identifier-first-flow-handler.md

```diff
@@ -0,0 +1,91 @@
+# Identifier-first Flow Handler
+
+The identifier-first login enables identifying the individuals prior to authenticating them. It is used to get the identity of the user without using authentication information, and use that identity to control the authentication flow.
+
+!!! info 
+    The identifier-first flow handler is shipped with the product itself from IS 5.7.0 onwards.
+
+## Configuring Identifier-first handler in the login flow
+
+!!! warning
+    This handler can be configured at any step in the authentication flow. However, it is not an authenticator by itself and needs to be configured along with another authenticator in order for the authentication process to be successful.
+
+1.  Make sure that the validate_username parameter in the IS_HOME/repository/conf/deployment.toml file is set to true if you want your username validated first.
+
+    ```toml
+    [authentication.authenticator.user_identifier] 
+    name ="IdentifierExecutor"
+    enable=true
+    [authentication.authenticator.user_identifier.parameters]
+    validate_username= true
+    ```
+2.  Log in to the WSO2 IS [management console](../../setup/getting-started-with-the-management-console/). Select a Service Provider (For details on how to create a new service provider, click [here](../../learn/adding-and-configuring-a-service-provider)) and expand the **Local & Outbound Authentication Configuration** section. Click **Advanced Configuration** and add the identifier-first as one of the steps as shown below. Since this handler does not work as an independant authenticator, you have to add another authentication step with the basic authenticator to save this configuration. 
```

Comment:
Can we break this into two steps? 

1. Add an authentication step and select `identifier-first`.

2. Add another authentication step and select `basic`. <insert explanation>